zerver/lib/cache_helpers.py

```diff
@@ -34,26 +38,33 @@ def message_fetch_objects():
                                                     id__gt=max_id - MESSAGE_CACHE_SIZE)
 
 def message_cache_items(items_for_remote_cache, message):
+    # type: (Dict[str, Any], Message) -> None
```

Comment:
Isn't this first argument a `Dict[str, Message]`?  Similarly for the next N items